modules/ROOT/pages/afm-add-schedule.adoc

```diff
@@ -5,43 +5,57 @@ endif::[]
 
 :imagesdir: ../assets/images
 
-You can add a schedule to an existing monitor if either of the following two conditions is true:
+You can add a schedule to an existing monitor if either of the following two conditions are true:
 
-* You plan to run the test associated with the schedule in a public location and the allocation of schedules for your organization in Anypoint Platform is not exhausted.
+* You plan to run the test associated with the schedule in a public location, and the allocation of schedules for your Anypoint Platform organization has not been exhausted.
 * You plan to run the test associated with the schedule in a private location.
 
-When you add a schedule to a monitor, you can use the schedule to run any of the versions of the test for that monitor.
-
 == Procedure
 
-There are two different methods that you can use to add a schedule.
-
-To add a schedule from the *Schedules* section of a monitor:
-
 . Select the monitor in which you want to add a schedule.
-. Click the *Schedules* tab.
-. Click *Add Schedule*.
-. In the Add Schedule dialog, specify this information:
+. Click *Add Schedule* from the *Schedules* table.
+. In the Add Schedule dialog, specify the following:
 +
-* The test version that you want the schedule to run
-* The location to run the test from
-* The frequency at which to run the test
-. Click *Add*.
+* The location in which to run the test
+* The interval at which to run the test:
++
+|===
+|Interval |Required Information |Defintion
 
-To add a schedule from the list of schedules that use a particular location:
+|Hourly
+|Minutes
+|The number of minutes after each hour when the test runs, beginning 60 minutes after the schedule has been created
```

Comment:
I have a couple of questions about the meaning and wording of this sentence.
Does "after each hour" refer to "clock hours" 1, 2, 3...or to 60 minutes passing?
Depending on the answer to that, does this mean that the test begins exactly 60 minutes after the schedule is created (what does that mean? What signifies the completion of the creation?) or that 60 minutes after creation, the test period begins but does not initiate until the specified number of minutes after "the hour" (whether that is the clock hour or the 60 minutes)?

Reply:
Maybe we need a SHORT example?

Reply:
In any case, ditto simple present tense:
```suggestion
|The number of minutes, after each hour, that triggers the test, beginning 60 minutes after the schedule is created.
```

Reply:
Ok cool, I reworded for clarity. Hopefully this is better:
"The minute at which the test begins, starting 60 minutes after you create the schedule"

I don't think we need to include an example since the "minutes" value is a list option in the UI.